.eslintrc.js

```diff
@@ -7,11 +7,10 @@ module.exports = {
       extends: ['@metamask/eslint-config-typescript'],
     },
     {
-      files: ['*.test.js'],
+      files: ['*.test.ts'],
       extends: ['@metamask/eslint-config-jest'],
     },
   ],
-  plugins: ['jsdoc'],
```

Comment:
This plugin is already added in our general shared config, so this was redundant.